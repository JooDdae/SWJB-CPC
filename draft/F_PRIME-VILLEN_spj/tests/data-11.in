390736378293977289770571074603123743
