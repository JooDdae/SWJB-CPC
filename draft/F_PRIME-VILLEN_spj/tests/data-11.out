18
2 155 509 145 2 25 4 1 24 4 907 28 623 0 1 2 4 14 